h/traversal/roots.py

```diff
@@ -283,7 +285,12 @@ def __init__(self, request):
         self.user_svc = self.request.find_service(name="user")
 
     def __getitem__(self, userid):
-        user = self.user_svc.fetch(userid)
+        try:
+            user = self.user_svc.fetch(userid)
+        except InvalidUserId as e:
+            # In this context we failed because the user provided a bad id
```

Comment:
I think this comment could be clarified—a "bad ID" could mean one that doesn't, say, correspond to a valid DB entity, but in this case it's specific to validation/formatting.

Reply:
Yep